src/Raven.Server/Documents/Handlers/Admin/AdminIndexHandler.cs

```diff
@@ -28,44 +29,65 @@ public class AdminIndexHandler : DatabaseRequestHandler
         [RavenAction("/databases/*/admin/indexes", "PUT", AuthorizationStatus.DatabaseAdmin, DisableOnCpuCreditsExhaustion = true)]
         public async Task Put()
         {
-            await PutInternal(validatedAsAdmin: true);
+            var isReplicatedQueryString = GetStringQueryString("is-replicated", required: false);
```

Comment:
what is wrong with GetBoolValueQueryString ?

Reply:
done